webpack/components/Table/TableWrapper.js

```diff
@@ -182,10 +180,10 @@ const TableWrapper = ({
             />
           </FlexItem>
         }
-        {!disableSearch && !resolvedStatusNoContent &&
+        {!disableSearch && !unresolvedStatusOrNoRows &&
```

Comment:
This change is causing the Search component to rerender when active filters are changed causing all API calls of Search component to retrigger..That's causing the test failures cause tests are expecting the call only once even if changing activeFilters in the test.

Reply:
Thanks for looking into this. Does it mean this change is not good? @sjha4